packages/input-number/src/input-number.vue

```diff
@@ -126,7 +124,7 @@
       },
       precision() {
         const { value, step, getPrecision } = this;
-        return Math.max(getPrecision(value), getPrecision(step));
+        return Math.max(getPrecision(value === undefined ? 0 : value), getPrecision(step));
```

Comment:
Why duplicate the `=== undefined` check here when it is already performed inside the `getPrecision` function?